runners/apex/pom.xml

```diff
@@ -184,6 +184,7 @@
             <configuration>
               <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
               <excludedGroups>
+                org.apache.beam.sdk.testing.FlattenWithHeterogeneousCoders,
```

Comment:
I'm guessing this isn't your change - I wonder if it's possible to not have these shown in the review. I wonder if using git pull --rebase is necessary to avoid this when merging changes from the master? I'll investigate a bit if I get time